src/Fetch.fs

```diff
@@ -358,6 +402,16 @@ module Types =
         | KeepAlive of bool
         | Signal of AbortSignal
 
+type Response with
+    [<Emit("Response.json($0, $1)")>]
```

Comment:
`status` can be optional so if it's not present Fable will generate wrong code: `Response.json(foo, )`, better to use the ellipsis so Fable skips the comma if there's only one argument: `Response.json($0...)`

Reply:
Oh nice, I was not aware of that thanks!